Note the pipeline is having the following issue with `py-matplotlib`'s `setup_build_environment` not being able to find the `qhull` dependency in the expected install directory.  

```
==> [2021-09-24-15:09:38.982360] '/home/software/radiuss/__spack_path_placeholder__/__spack_path_placeholder__/__spack_path_placeholder__/__spack_path_placeholder__/__spack_path_placeholder__/__spack_path_placeholder__/__spack_path_placeholder__/__spack_path_placeholder__/__spack_path_placeholder__/__spack_path_placeholder__/__spack_path_placeholder__/__spack_path_placeholder__/__spack_path_placeholder__/__spack_path_placeholder__/__spack_path_placeholder__/__spack_path_placeholder__/__spack_path_placeholder__/__spack_path_placeholder__/__s/linux-ubuntu18.04-x86_64/gcc-7.5.0/python-3.8.11-zrvbe5ted5m4dx5lc5dlj33zlmzbiznz/bin/python3.8' '-c' '
import json
from distutils.sysconfig import (
    get_config_vars,
    get_config_h_filename,
    get_makefile_filename,
    get_python_inc,
    get_python_lib,
)

config = get_config_vars()
config['"'"'config_h_filename'"'"'] = get_config_h_filename()
config['"'"'makefile_filename'"'"'] = get_makefile_filename()
config['"'"'python_inc'"'"'] = {}
config['"'"'python_lib'"'"'] = {}

for plat_specific in [True, False]:
    plat_key = str(plat_specific).lower()
    config['"'"'python_inc'"'"'][plat_key] = get_python_inc(plat_specific, prefix='"'"''"'"')
    config['"'"'python_lib'"'"'][plat_key] = {}
    for standard_lib in [True, False]:
        lib_key = str(standard_lib).lower()
        config['"'"'python_lib'"'"'][plat_key][lib_key] = get_python_lib(
            plat_specific, standard_lib, prefix='"'"''"'"'
        )

print(json.dumps(config))
'
==> [2021-09-24-15:09:39.321253] No libraries found for qhull
==> [2021-09-24-15:09:39.805546] Error: NoLibrariesError: Unable to recursively locate qhull libraries in /home/software/radiuss/__spack_path_placeholder__/__spack_path_placeholder__/__spack_path_placeholder__/__spack_path_placeholder__/__spack_path_placeholder__/__spack_path_placeholder__/__spack_path_placeholder__/__spack_path_placeholder__/__spack_path_placeholder__/__spack_path_placeholder__/__spack_path_placeholder__/__spack_path_placeholder__/__spack_path_placeholder__/__spack_path_placeholder__/__spack_path_placeholder__/__spack_path_placeholder__/__spack_path_placeholder__/__spack_path_placeholder__/__s/linux-ubuntu18.04-x86_64/gcc-7.5.0/qhull-2020.2-44gutkll3zxrc5ebj25fzyqeka4jnp4x

/builds/spack/spack/var/spack/repos/builtin/packages/py-matplotlib/package.py:169, in setup_build_environment:
        166        for dep in self.spec.dependencies(deptype='link'):
        167            query = self.spec[dep.name]
        168            include.extend(query.headers.directories)
  >>    169            library.extend(query.libs.directories)
        170
        171        # Build uses a mix of Spack's compiler wrapper and the actual compiler,
        172        # so this is needed to get parts of the build working.


Traceback (most recent call last):
  File "/builds/spack/spack/lib/spack/spack/build_environment.py", line 1028, in _setup_pkg_and_run
    context=context)
  File "/builds/spack/spack/lib/spack/spack/build_environment.py", line 786, in setup_package
    pkg.setup_build_environment(env)
  File "/builds/spack/spack/var/spack/repos/builtin/packages/py-matplotlib/package.py", line 170, in setup_build_environment
  File "/builds/spack/spack/lib/spack/spack/spec.py", line 959, in __get__
    value = f()
  File "/builds/spack/spack/lib/spack/spack/spec.py", line 951, in <lambda>
    callbacks_chain.append(lambda: self.default(self, instance, cls))
  File "/builds/spack/spack/lib/spack/spack/spec.py", line 890, in _libs_default_handler
    raise spack.error.NoLibrariesError(msg.format(spec.name, spec.prefix))
spack.error.NoLibrariesError: Unable to recursively locate qhull libraries in /home/software/radiuss/__spack_path_placeholder__/__spack_path_placeholder__/__spack_path_placeholder__/__spack_path_placeholder__/__spack_path_placeholder__/__spack_path_placeholder__/__spack_path_placeholder__/__spack_path_placeholder__/__spack_path_placeholder__/__spack_path_placeholder__/__spack_path_placeholder__/__spack_path_placeholder__/__spack_path_placeholder__/__spack_path_placeholder__/__spack_path_placeholder__/__spack_path_placeholder__/__spack_path_placeholder__/__spack_path_placeholder__/__s/linux-ubuntu18.04-x86_64/gcc-7.5.0/qhull-2020.2-44gutkll3zxrc5ebj25fzyqeka4jnp4x
==> [2021-09-24-15:09:39.823619] Flagging py-matplotlib-3.4.3-hlowcb2pgat2nk23nh3sqzwrx2j4opzn as installed
==> [2021-09-24-15:09:39.823763] Downgrading to a read lock on py-matplotlib-3.4.3-hlowcb2pgat2nk23nh3sqzwrx2j4opzn with timeout 1e-09
```

(See https://gitlab.spack.io/spack/spack/-/jobs/1047942/raw for full details.)

--- a/var/spack/repos/builtin/packages/qhull/package.py
+++ b/var/spack/repos/builtin/packages/qhull/package.py
@@ -19,7 +19,6 @@ class Qhull(CMakePackage):
     homepage = "http://www.qhull.org"
     url = "https://github.com/qhull/qhull/archive/refs/tags/2020.2.tar.gz"
 
-    version('2020.2', sha256='59356b229b768e6e2b09a701448bfa222c37b797a84f87f864f97462d8dbc7c5')
     version('2020.1', sha256='0258bbf5de447e3d6b3968c5a7b51c08ca5d98f11f94f86621ed3e7c98365b8d')
     version('2019.1', sha256='cf7235b76244595a86b9407b906e3259502b744528318f2178155e5899d6cf9f')
     version('2015.2', sha256='8b6dd67ff77ce1ee814da84f4134ef4bdce1f1031e570b8d83019ccef58b1c00')

